loader.rb

```diff
@@ -83,7 +83,7 @@
 
 AUTOLOAD_CONSTANTS.freeze
 
-if Config.production?
+if Config.production? || Config.e2e_test?
```

Comment:
I agree that we should keep the E2E and production environments as similar as possible. I believe that an extra environment with `Config.e2e_test?` can lead to divergence between the E2E and production environments, and encouraging people. If we decide to use blob storage for default images in production, I think we should remove `Config.e2e_test?` and strive to set `RACK_ENV=production` for E2E tests.

ref: https://github.com/ubicloud/ubicloud/pull/1664#discussion_r1630704271

Reply:
Resolving it, as we'll be considering to do that with subsequent PR.